### PR TITLE
Inclusion of integrators to common_controls

### PR DESCRIPTION
**Pull request by tavovalmo**
I have included two integrators with windup and anti-windup limiters.

### CODE REVIEW

**Comment by jd-lara** [comment]
@tavovalmo thanks for the contribution. We are just missing unit tests for these integrators. Do you mind adding them?

File: src/models/common_controls.jl [hunk 1]
```diff
@@ -1,6 +1,88 @@
...
+    u::Z,
+    y::Z,
+    K::Float64,
+    T::Float64,
+    y_min::Float64,
+    y_max::Float64
+)  where {Z <: PSID.ACCEPTED_REAL_TYPES}
+    dydt_scaled = K * u
+    y_sat = clamp(y, y_min, y_max)
+    binary_logic = ((y >= y_max) && (dydt_scaled > 0)) || ((y <= y_min) && (dydt_scaled < 0)) ? 0.0 : 1.0
```
Comment (jd-lara) [inline src/models/common_controls.jl:67]: @rodrigomha we need to check if this syntax is type stable

**Review by rodrigomha — CHANGES_REQUESTED** [review]
It is not required to have the PSID part in PSID.ACCEPTED_REAL_TYPES.

Regarding the tests, it may be not required for now if we plan to add them in a new AVR model. Alternatively we can add them in test_base to test type stability.

**Comment by jd-lara** [comment]
@tavovalmo why did you close this PR? it just required some tests to be merged.

**Comment by tavovalmo** [comment]
I eliminated the branch after I merged the changes in my remote repo. It seems that this action closed the PR. Sorry. I will create a new one.

**Pull request closed without merging**